runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/WeaponType.java

```diff
@@ -27,45 +27,47 @@
 import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.ACCURATE;
+import static net.runelite.client.plugins.attackstyles.AttackStyle.ACCURATE_RANGING;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.AGGRESSIVE;
+import static net.runelite.client.plugins.attackstyles.AttackStyle.AIM_AND_FIRE;
+import static net.runelite.client.plugins.attackstyles.AttackStyle.BLOCK;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.CASTING;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.CONTROLLED;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.DEFENSIVE;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.DEFENSIVE_CASTING;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.LONGRANGE;
-import static net.runelite.client.plugins.attackstyles.AttackStyle.OTHER;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.RANGING;
 
 enum WeaponType
 {
-	TYPE_0(ACCURATE, AGGRESSIVE, null, DEFENSIVE),
-	TYPE_1(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_2(ACCURATE, AGGRESSIVE, null, DEFENSIVE),
-	TYPE_3(RANGING, RANGING, null, LONGRANGE),
-	TYPE_4(ACCURATE, AGGRESSIVE, CONTROLLED, DEFENSIVE),
-	TYPE_5(RANGING, RANGING, null, LONGRANGE),
-	TYPE_6(AGGRESSIVE, RANGING, CASTING, null),
-	TYPE_7(RANGING, RANGING, null, LONGRANGE),
-	TYPE_8(OTHER, AGGRESSIVE, null, null),
-	TYPE_9(ACCURATE, AGGRESSIVE, CONTROLLED, DEFENSIVE),
-	TYPE_10(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_11(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_12(CONTROLLED, AGGRESSIVE, null, DEFENSIVE),
-	TYPE_13(ACCURATE, AGGRESSIVE, null, DEFENSIVE),
-	TYPE_14(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_15(CONTROLLED, CONTROLLED, CONTROLLED, DEFENSIVE),
-	TYPE_16(ACCURATE, AGGRESSIVE, CONTROLLED, DEFENSIVE),
-	TYPE_17(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_18(ACCURATE, AGGRESSIVE, null, DEFENSIVE, CASTING, DEFENSIVE_CASTING),
-	TYPE_19(RANGING, RANGING, null, LONGRANGE),
-	TYPE_20(ACCURATE, CONTROLLED, null, DEFENSIVE),
-	TYPE_21(ACCURATE, AGGRESSIVE, null, DEFENSIVE, CASTING, DEFENSIVE_CASTING),
-	TYPE_22(ACCURATE, AGGRESSIVE, AGGRESSIVE, DEFENSIVE),
-	TYPE_23(CASTING, CASTING, null, DEFENSIVE_CASTING),
-	TYPE_24(ACCURATE, AGGRESSIVE, CONTROLLED, DEFENSIVE),
-	TYPE_25(CONTROLLED, AGGRESSIVE, null, DEFENSIVE),
-	TYPE_26(AGGRESSIVE, AGGRESSIVE, null, AGGRESSIVE),
-	TYPE_27(ACCURATE, null, null, OTHER);
+	TYPE_0(ACCURATE, AGGRESSIVE, null, DEFENSIVE), //Unarmed
```

Comment:
It's not important to maintain the `TYPE_0` naming scheme. Instead, rename the enums themselves.